services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from django.db.models import QuerySet, DateTimeField
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: DateTimeField,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),

```

Comment:
movie_id=movie_id
1) less queries to db
2) more readable

read item 1 here: https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency

Reply:
When i use movie_id=movie_id
Pytest gives me an error 
django.core.exceptions.FieldError: Cannot resolve keyword 'movie_id' into field. Choices are: actors, description, genres, id, moviesession, title

Reply:
done